standard/docs/en/schema/reference.md

```diff
@@ -395,7 +395,9 @@ The items block is used to list the line-items associated with a tender, award o
 ```
 #### Unit
 
-The `unit` block allows detailed specification of the parameters and price of units that make up a line-item. Although no codelist for units has been established in the current release of the standard, publishers may consider using the Units provided by the [Quantities, Units, Dimensions and Data Types Ontologies](http://www.qudt.org/qudt/owl/1.0.0/unit/) in the `unit.name` field (drawing on the CamelCase unit names, such as SquareMile), in order to provide detailed information the cost per unit of a line-item. 
+The `unit` block allows detailed specification of the parameters and price of units that make up a line-item.
+
+If the [Quantities, Units, Dimensions and Data Types Ontologies](http://www.qudt.org/qudt/owl/1.0.0/unit/) unit classification scheme is used, then publishers can use its CamelCase unit names, such as "SquareMile", in the `unit.name` field.
```

Comment:
I suggest adding an extra sentence to make it clear that other unit classification schemes can be used.

> Other unit classification schemes can be used and alternatives are listed in  the [unit classification scheme codelist](http://standard.open-contracting.org/latest/en/schema/codelists/#unit-classification-scheme).

Reply:
Fixed in c2b7578 with slightly different wording